benchmark/benchmark.py

```diff
@@ -152,7 +154,7 @@ def build_inputs_chatglm(tokenizer, query: List[str], padding, history: List[Tup
         Next_token_throughput = 1000 / latency_90 * args.batch_size
         print("\n")
         print("=" * 50 + args.model_name + " Final Performance" + "=" * 50)
-        print(f"First token Latency:\t{first_token_latency:.2f} ms")
+        print(f"Input token size: {input_token_nums}, First token Latency:\t{first_token_latency:.2f} ms")
```

Comment:
The input token length has been printed in Line 113.

Reply:
Maybe we can remove line113 since we usually use grep to process logs, and we care about the first token latency, the performance of which depends on the input token size.
This makes it easy to see the cause of the expected performance mismatch between two different models with the same input.

Reply:
Batch size is also important for the 1st token, printed in line 114. It's okay to move these two lines here.

Reply:
yes, agree with you.  :)